docs/pages/components/tile.md

```diff
@@ -10,186 +10,637 @@ summary:
-    </div>
-    <div class="fd-tile__content">
-        <h3 class="fd-tile__title">Tile Title</h3>
-    </div>
-</div>
+## Launch Tile
+
+Structure of the Numeric Content
+<ul class="docs-ul">
+    <li>Launch Icon Container - <code>fd-numeric-content__launch-icon-container</code>
+        <ul class="docs-ul">
+            <li>Launch Icon - <code>fd-numeric-content__launch-icon sap-icon--line-charts</code></li>
+        </ul>
+    </li>
+    <li>KPI Container <strong>(optional)</strong> - <code>fd-numeric-content__kpi-container</code>
+        <ul class="docs-ul">
+            <li>KPI <strong>(optional)</strong> - <code>fd-numeric-content__kpi</code>
+                <ul class="docs-ul">
+                    <li>States:
+                        <ul class="docs-ul">
+                            <li>neutral (default)</li>
+                            <li>positive - <code>fd-numeric-content__kpi--positive</code></li>
+                            <li>negative - <code>fd-numeric-content__kpi--negative</code></li>
+                            <li>critical - <code>fd-numeric-content__kpi--critical</code></li>
+                            <li>informative - <code>fd-numeric-content__kpi--informative</code></li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+    <li>Scale Container <strong>(optional)</strong> - <code>fd-numeric-content__scale-container</code>
+        <ul class="docs-ul">
+            <li>Scale <strong>(optional)</strong> - <code>fd-numeric-content__scale</code>
+                 <ul class="docs-ul">
+                    <li>States:
+                        <ul class="docs-ul">
+                            <li>neutral (default)</li>
+                            <li>positive - <code>fd-numeric-content__scale--positive</code></li>
+                            <li>negative - <code>fd-numeric-content__scale--negative</code></li>
+                            <li>critical - <code>fd-numeric-content__scale--critical</code></li>
+                            <li>informative - <code>fd-numeric-content__scale--informative</code></li>
+                        </ul>
+                    </li>
+                    <li>Deviation Arrows <strong>(optional)</strong>: <code>sap-icon--down</code> or <code>sap-icon--up</code></li>
+                    <li>Text <strong>(optional)</strong>: <code>fd-numeric-content__scale-text</code></li>
+                 </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+<br>
+Numeric Content Sizes:
+<ul class="docs-ul">
+    <li>L-Numeric Content size. Longest combination for 4-character languages</li>
+    <li>M-Numeric Content size. Longest combination for 6-character languages</li>
+    <li>S-Numeric Content size. Longest combination for 8-character languages</li>
+</ul>
 
 <br>
+The header contains a maximum of 3 lines title or a maximum of 2 lines title with 1 line subtitle.
```

Comment:
The header can contain a maximum of 3 lines; all 3 lines can be used for the title or one of the lines can be used for a subtitle. If a 4 line title or an optional subtitle line is required, the `fd-tile_header-content` can be used to represent a header and will merge the content containers into one tile.